integration/gethnetwork/geth_network_test.go

```diff
@@ -1,44 +1,55 @@
 package gethnetwork
 
 import (
+	"errors"
 	"fmt"
-	"os/exec"
+	"math/big"
 	"strconv"
 	"testing"
+	"time"
+
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/obscuronet/obscuro-playground/go/ethclient"
+	"github.com/obscuronet/obscuro-playground/integration/datagenerator"
+	"gopkg.in/yaml.v3"
 )
 
 const (
-	shCmd          = "sh"
-	gethBinarySh   = "./build_geth_binary.sh"
-	gethBinaryPath = "./geth-release-1.10.17"
-
 	numNodes        = 3
-	expectedChainID = "777"
+	expectedChainID = "1337"
 
 	peerCountCmd = "net.peerCount"
 	chainIDCmd   = "admin.nodeInfo.protocols.eth.config.chainId"
 )
 
-func init() { //nolint:gochecknoinits
-	_, err := exec.Command(shCmd, gethBinarySh).Output()
+var timeout = 15 * time.Second
+
+func TestAllNodesJoinSameNetwork(t *testing.T) {
+	gethBinaryPath, err := EnsureBinariesExist(LatestVersion)
```

Comment:
I think this method might be better named, "FindGethBinaries" or similar.

Reply:
The `EnsureBinariesExist` ?  It downloads and builds them too and I wouldn't add again geth since it's already in the `gethnetwork` package. 🤔

Reply:
@otherview My main point is that the name implies it's just an assertion, but it actually also does the work of getting the binaries. No big deal.